### ISSUE TITLE
Weblinks Ordering - Add Most Recent ASC/DESC

### ISSUE THREAD

**Issue by modusworks** [joomla-extensions/weblinks#480]
Hello, 

We quite need the ability in both the component and module to set weblink ordering by date (ascending / descending) whereas now ordering must be done manually (drag/drop). It'd be great to set this globally or at the menu item / module parameters. 

This is a needed function for most sites that ultilize weblinks for 3rd party news or resources. 

Thanks Team!

**Comment by joomdonation** [comment]
This makes sense and should be doable. We can add a new setting (available as Global and can be changed in menu item parameter) to allow choosing ordering of weblinks:

- Title
- Created Date
- Modified Date
- Hits
- Ordering (default)

Will try to find sometime to implement it.

**Comment by modusworks** [comment]
Awesome! 

This will be an excellent feature and make the component more flexible. Could we also implement to the associated weblinks module? 

Thanks!

**Comment by joomdonation** [comment]
> Could we also implement to the associated weblinks module

Yes, of course. When we support this in the component, we should also support it in module

**Comment by Pinkeltje** [comment]
Any news on this?

**Comment by Pinkeltje** [comment]
Same for categories when using menu-item List All Web Link Categories

**Comment by Pinkeltje** [comment]
Is there still any activity on weblinks development? No answers on issues since a long time!
